build.gradle

```diff
@@ -1,19 +1,21 @@
 buildscript {
     ext {
-        compose_version = '1.3.0-beta02'
+        compose_version = '1.5.0-alpha02'
     }
     repositories {
         google()
         mavenCentral()
     }
-    dependencies {
-        classpath "com.android.tools.build:gradle:7.2.2"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10"
-        classpath "com.google.dagger:hilt-android-gradle-plugin:2.43.2"
-        classpath 'com.google.gms:google-services:4.3.13'
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.1'
-        classpath 'com.google.firebase:perf-plugin:1.4.1'
-    }
+}
+
+plugins {
+    id 'com.android.application' version '8.0.0' apply false
+    id 'com.android.library' version '8.0.0' apply false
+    id 'org.jetbrains.kotlin.android' version '1.8.0' apply false
```

Comment:
Version 1.8.20 is available. Can we use that instead?

Reply:
Done!